openmetadata-service/src/main/java/org/openmetadata/service/Entity.java

```diff
@@ -91,7 +91,7 @@ public final class Entity {
   public static final String MLMODEL = "mlmodel";
   public static final String CONTAINER = "container";
   public static final String BOT = "bot";
-  public static final String ALERT = "alert";
+  public static final String EVENT_SUBSCRIPTION = "eventSubscription";
```

Comment:
Is there any other kind of subscription possible other than an Event Subscription? If so, we should just call this subscription.